apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.tsx

```diff
@@ -37,7 +37,36 @@ const displayAddresses = (result: PatientSearchResult): JSX.Element => (
     </div>
 );
 
-const displayPhones = (result: PatientSearchResult): string => result.phones.join('\n');
+const phoneTypeResolver = (typeCode?: string) => {
+    switch (typeCode) {
+        case 'PH':
+            return 'Phone';
+        case 'AN':
+            return 'Answering service';
+        case 'BP':
+            return 'Beeper';
+        case 'CP':
+            return 'Cellular phone';
+        case 'NET':
+            return 'Email address';
+        case 'FAX':
+            return 'Fax';
+        default:
+            return typeCode;
+    }
+};
```

Comment:
Issue: These values should be pulled from the value set

Reply:
these values should be coming from the API response